tests/tests.js

```diff
@@ -139,23 +137,23 @@ exports.defineAutoTests = function() {
         });
     });
 };
-exports.defineManualTests = function(contentEl, createActionButton) {
+exports.defineManualTests = function (contentEl, createActionButton) {
 
-    createActionButton('Listen to orientationchange events', function() {
-        window.addEventListener("orientationchange", function(){
-            contentEl.innerHTML += '<p>Orientation changed! ' + screen.orientation.type + '</p>';
+    createActionButton('Listen to orientationchange events', function () {
+        window.addEventListener('orientationchange', function () {
+            contentEl.innerHTML += '<p>Orientation changed! ' + window.screen.orientation.type + '</p>';
```

Comment:
functional change?

Reply:
> What are all these functional changes here that add `window.` before existing code?

It does not change functionality. `screen` is attached to the `window` object. On the other hand, `eslint` already know the `windows` object, but does not know `screen`. It is better to append `window` on `screen`, so people understand that it's from the `window` object. It is also possible to add `screen` as an exception, but I would prefer the proper variable usage.

Reply:
+1 for adding `window.` to resolve the eslint issue.